/var/tmp is read-only under rootbld (and also a weird place to assume for tmp)

diff --git a/ssl-cert-check b/ssl-cert-check
index 2bb088a..0a98744 100755
--- a/ssl-cert-check
+++ b/ssl-cert-check
@@ -885,8 +885,8 @@ else
 fi
 
 # Place to stash temporary files
-CERT_TMP=$($MKTEMP /var/tmp/cert.XXXXXX)
-ERROR_TMP=$($MKTEMP /var/tmp/error.XXXXXX)
+CERT_TMP=$($MKTEMP cert.XXXXXX)
+ERROR_TMP=$($MKTEMP error.XXXXXX)
 
 ### Baseline the dates so we have something to compare to
 MONTH=$(${DATE} "+%m")
